pennylane_lightning/src/gates/cpu_kernels/GateImplementationsLM.hpp

```diff
@@ -24,6 +24,7 @@
 #include "LinearAlgebra.hpp"
 #include "PauliGenerator.hpp"
 
+#include <bit>
```

Comment:
I think we don't need to include this here or even in `GateImplementationsPI.hpp`. right?
```suggestion
```